lawlibrary/settings.py

```diff
@@ -0,0 +1,7 @@
+from peachjam.settings import *  # noqa
```

Comment:
Since lawlibrary builds on liiweb, perhaps change this to `from liiweb.settings import *`. That gives liiweb a chance to set appropriate defaults.

You'd then only need to add `lawlibrary` to `INSTALLED_APPS`, and not `liiweb`.